.github/workflows/stylua.yml

```diff
@@ -0,0 +1,14 @@
+name: Stylua
```

Comment:
You will have to set this up to run as a workflow in the repo's settings.

Reply:
It's just had a thought that you may(?) not have done this before. Here's a helpful doc :) 

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#enabling-workflows-for-forks-of-private-repositories